packages/gatsby-transformer-react-docgen/src/Doclets.js

```diff
@@ -32,7 +32,8 @@ const getDoclets = str => {
 
   for (; match; match = DOCLET_PATTERN.exec(str)) {
     val = match[2] ? match[2].replace(/\r$/, ``) : true
-    doclets[match[1]] = val
+    const key = match[1]
+    doclets[key] = doclets[key] ? [].concat(doclets[key], val) : val
```

Comment:
Will this potentially cause type conflicts during schema inferring? For given node type, that field can be either single value or array

Reply:
it shouldn't, these are typed as JSON now-a-days

Reply:
![screenshot 2018-12-07 at 17 43 45](https://user-images.githubusercontent.com/419821/49660787-b91b5b80-fa47-11e8-92f8-d0767c69c3e2.png)
So we still show false warning - seems like the ignore thingy stopped working (or didn't work for deep paths). I'll take a look on that.

I don't really like putting more handling of data shape on the users (sometimes property will be single, sometimes array), but unless we do something like define which properties should be arrays globally there's not much we can do about it I guess

Reply:
Agh, we broke it recently - will fix and add some tests to catch regressions, so as far as this PR goes, I think it's good to go. But I do think we need to bump major version here as this will potentially break some sites

Reply:
yes major bump sounds good 👍